src/components/list-items/list-items.css

```diff
@@ -1,9 +1,12 @@
 .list-items {
-  margin: 20px 10px 0 0;
+  padding: 20px 0px 0px 20px;
+  margin-left: 20px;
   font-size: 20px;
 }
 
 .check-box {
   display: flex;
```

Comment:
I don't think that flex column is needed here
div is a block element, so it's always starting from the new line and takes the whole length of the parent container